sorc/IMSaggregate_mod.f90

```diff
@@ -359,15 +389,15 @@ subroutine write_IMS_outputs_vec(idim, jdim, otype, date_str,scfIMS, sndIMS, lon
     call netcdf_err(error, 'defining oro long name' )
 
     !--- define snow cover
-    error = nf90_def_var(ncid, 'IMSscf', nf90_double, id_obs, id_scfIMS)
+    error = nf90_def_var(ncid, 'IMSscf', nf90_double, id_scfIMS)
```

Comment:
Don't we need to define dimensions even if it is 1D?

Reply:
@CoryMartin-NOAA, I double check the original code from NOAA-PSL, YES, it is like location in ioda converter. As it is a variable, it needs a dimension.

Reply:
Ok please add that back in where appropriate